### PR TITLE
Replaced titangraph dependencies with janusgraph equivalents.

### PR DESCRIPTION
**Pull request by DarioAmiri**
Long live [JanusGraph](http://janusgraph.org/).

### CODE REVIEW

File: service/src/test/java/org/eclipse/keti/acs/privilege/management/dao/GraphResourceRepositoryTest.java [hunk 1]
```diff
@@ -92,7 +92,7 @@ public void teardown() {
     }
 
     private void setupTitanGraph() throws InterruptedException, ExecutionException {
```
[the remark this answers is not in the document]
Comment (DarioAmiri) [reply service/src/test/java/org/eclipse/keti/acs/privilege/management/dao/GraphResourceRepositoryTest.java]: Basically, I just wanted to test the waters before renaming things left and right. I can replace all references to Titan in variables in methods, but do you want me to replace the references in the POM as well?

**Comment by DarioAmiri** [comment]
I added a commit that introduces more extensive removal of references to titan. Let me know if you approve. I can also squash the commits if necessary.

**Comment by anubhavi25** [comment]
@DarioAmiri the changes look good to me. Yes please squash commits and I'll merge it in.

**Comment by DarioAmiri** [comment]
@anubhavi25 I squashed the PR commits.

**Pull request merged**